src/app/pages/system-variables/system-variables.html

```diff
@@ -0,0 +1,242 @@
+<p>
+  Angular Material components depend on system variables defined as CSS variables through the
+  <span class="demo-surface-variable">material.theme</span>
+  Sass mixin. This page provides guidance and documentation for using these variables to
+  customize components.
+</p>
+
+<h2 id="colors">Colors</h2>
+
+<p>
+  Material Design uses color to create accessible, personal color schemes
+  that communicate your product's hierarchy, state, and brand. See Material
+  Design's <a href="https://m3.material.io/styles/color/system/overview">Color System</a>
+  page to learn more about its use and purpose.
+</p>
+<p>
+  The following colors are the most often used in Angular Material components. Use these
+  colors and follow their uses to add theme colors to your application's custom components.
+</p>
+
+<div class="demo-group">
+  <div class="demo-color-container">
+    <div class="demo-heading"
+         [style.background-color]="'var(--mat-sys-primary)'"
+         [style.color]="'var(--mat-sys-on-primary)'">
```

Comment:
These `style` bindings aren't doing anything. You can either use a static attribute (`style="background-color: var(--mat-sys-primary)"`) or move them into the stylesheet. Same applies to the ones further down.

Reply:
Ah yeah, I was planning to make this more programatic but it ended up getting over-engineered